static/styles/rendered_markdown.scss

```diff
@@ -141,6 +141,19 @@
         vertical-align: text-bottom;
     }
 
+    /* Timestamps */
+    .timestamp {
+        background: hsl(0, 0%, 93%);
+        border-radius: 3px;
+        padding: 0 0.2em;
+        box-shadow: 0px 0px 0px 1px hsl(0, 0%, 80%);
+        white-space: nowrap;
+        margin-left: 2px;
+        margin-right: 2px;
+        display: inline-block;
+        margin-bottom: 1px;
+    }
```

Comment:
Please add a commit adding docs for the feature, before we forget.

Reply:
Resolved in #15054.